opm/simulators/linalg/PreconditionerFactory.hpp

```diff
@@ -134,6 +134,10 @@ class PreconditionerFactory
         criterion.setNoPreSmoothSteps(prm.get<int>("pre_smooth", 1));
         criterion.setNoPostSmoothSteps(prm.get<int>("post_smooth", 1));
         criterion.setDebugLevel(prm.get<int>("verbosity", 0));
+        // As the default we request to accumulate data to 1 process always as our matrix
+        // graph might be unsymmetric and hence not supported by the PTScotch/ParMetis
+        // calls in DUNE. Accumulating to 1 skips PTScotch/ParMetis
+        criterion.setAccumulate(static_cast<Dune::Amg::AccumulationMode>(prm.get<int>("accumulate", 1)));
```

Comment:
Which accumulation mode is "1" cast to? I assume it is atOnceAccum? If so, does that mean that we go from N to 1 process immediately on level 2, or that we coarsen several levels on N processes, and then go to 1 process? I will assume the second option, which seems the most reasonable. This would be the same as the "no PTScotch/ParMETIS at all" way, I think?

Perhaps we should use a string parameter instead of an integer?

Reply:
It casts to atOnceAccu as the other would trigger the same problem (pass an unsymmetric graph to a software assuming a symmetric one).